Automattic/wp-calypso
# Subscriptions: Add the Subscriber Login navigation placement toggle

Related to peKye1-EA-p2

## Proposed Changes

It adds the Subscriber Login navigation placement toggle.

<img width="737" alt="Screenshot 2024-04-29 at 14 48 26" src="https://github.com/Automattic/wp-calypso/assets/4068554/e5f6f361-289a-43dd-b9f3-46964faab21a">
<br><br>

Blocked by https://github.com/Automattic/jetpack/pull/37123

## Testing Instructions

1. Go to Newsletter Settings
2. Make sure the toggle and "Preview and edit" link works as expected

## Pre-merge Checklist
